docs/local-deploy.md

```diff
@@ -0,0 +1,112 @@
+[![Travis CI status](https://api.travis-ci.org/CatalystCode/project-fortis-spark.svg?branch=master)](https://travis-ci.org/CatalystCode/project-fortis-spark)
```

Comment:
I'd move this to `docs/developer/local-deploy.md` since the developer-centric docs have recently been moved there.